.github/workflows/sync.yml

```diff
@@ -0,0 +1,40 @@
+name: 'Sync with MUX'
+
+on:
+  schedule:
+  - cron: '0 0 1 * *'
```

Comment:
This runs monthly, with ability to manual run from "Actions" tab on GitHub. Let me know if you want it to be more frequent.

Reply:
I think this will run quite quickly, so it could even be weekly